examples/cc2538-common/Makefile

```diff
@@ -0,0 +1,11 @@
+CONTIKI_PROJECT = test-pwm timer-test
+
+ifndef TARGET
+TARGET = cc2538dk
+endif
```

Comment:
Yes, please. I would not care much apart from the fact that it's unusual in Contiki. But I've just tested it, and it's broken:
```
make TARGET=zoul savetarget
make BOARD=firefly
```
builds for `cc2538dk` instead of `zoul` because the `TARGET` defined in `Makefile` takes precedence over the one defined in `Makefile.target`, because `Makefile.include`, which includes `Makefile.target`, is included at the end of `Makefile`, after your definition of `TARGET` to `cc2538dk`. But `make TARGET=zoul BOARD=firefly` works as expected because in this case `TARGET` is defined to `zoul` right from the beginning, so your definition does not override it.